Src/StartMenu/StartMenuDLL/StartMenuDLL.rc

```diff
@@ -1181,6 +1188,7 @@ BEGIN
     IDS_STRING7108          "Midnight skin\n\nSkin with dark background.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
     IDS_STRING7109          "Metro skin\n\nSkin that uses the start screen colors.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
     IDS_STRING7110          "Metallic skin\n\nA start menu skin with metallic look.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2009-2017, Ivo Beltchev"
+    IDS_STRING7111          "Immersive skin\n\nA skin that uses immersive colors and modern visual elements in the style of Windows 10.\n\nPart of <A HREF=""https://github.com/Open-Shell/Open-Shell-Menu/"">Open-Shell</A> (c) 2023, The Open-Shell Team"
```

Comment:
Shouldn’t the copyright be the same as the other skins? Unless it’s referring to the creation of the skin itself and not the program.

Also, I think it would make sense for the skin copyright to read `(c) 2017-2018`, as it’s consistent with Open-Shell and not Classic Shell.

Maybe I’m not quite sure about the proper usage.

Reply:
> Shouldn’t the copyright be the same as the other skins? Unless it’s referring to the creation of the skin itself and not the program.

The copyright is related to the skin itself. And since this is new skin added by `Open-Shell Team` it shouldn't contain copyright from the original skins.

> Also, I think it would make sense for the skin copyright to read (c) 2017-2018, as it’s consistent with Open-Shell and not Classic Shell.

I'm not sure about that too.
From what I read the `X-Y` form is used to denote copyright claim in years X until Y. And the `X` form means copyright claim from year X onwards. But I'm no expert in these things.

And I'm not even sure we need to care about this (as anybody is free to use whatever part of `Open-Shell` as he likes).

Reply:
https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work

According to this, copyright notice should list years when given _code_ was changed.
So using `(c) 2023, The Open-Shell Team` seems to be correct for newly added Immersive skin.

Reply:
Another nice blog post:
https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code

Reply:
Ok, that makes sense.